relayer/chains/icon/route.go

```diff
@@ -12,7 +13,13 @@ import (
 )
 
 func (p *Provider) Route(ctx context.Context, message *providerTypes.Message, callback providerTypes.TxResponseFunc) error {
-	p.log.Info("starting to route message", zap.Any("message", message))
+	p.log.Info("starting to route message",
+		zap.Any("sn", message.Sn),
+		zap.Any("req-id", message.ReqID),
+		zap.String("src", message.Src),
+		zap.String("event-type", message.EventType),
```

Comment:
can you maintain consistency and use "_" instead of "-" for fields.

Reply:
done

Reply:
Please don't use any. It slows things.

Reply:
Sometimes, req-id is nil and in those cases we need to handle if it is nil or not. I think we can compromise on this. How much slow will it be by using Any. It is not noticeable.

Reply:
I understand, I believe even if it is nil it will be handled accordingly.

Reply:
yes if we use zap.Any().